README.md

```diff
@@ -5,6 +5,16 @@ Keycloak is an Open Source Identity and Access Management solution for modern Ap
 This repository contains the source code for the Keycloak Server, Java adapters and the JavaScript adapter.
 
 
+## moneymeets Specific Fixes
+
+At moneymeets the repository keycloak-docker is used to set up and run Keycloak. Internally, it creates a docker container containing Keycloak based on the source of the latest commit in branch moneymeets/20.0.5 of this repository here.
+This fork was created so that moneymeets is able to implement fixes that are not available yet or only available in later Keycloak versions.
+Currently, the following fixes are present in the code:
+
+* Back port of fix [keycloak/keycloak#23303](https://github.com/keycloak/keycloak/pull/23303).
+* MD-6730 fix email validation to be RFC compliant.
```

Comment:
🔥 great to have some overview of why we have this fork at all. I hope it's up to date and can also be seen via git commit history?